Package.swift

```diff
@@ -10,7 +10,7 @@ let package = Package(
         .library(name: "AlamofireSessionRenewer", targets: ["AlamofireSessionRenewer"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/Alamofire/Alamofire.git", .upToNextMajor(from: "5.4.3"))
+        .package(url: "https://github.com/Alamofire/Alamofire.git", .upToNextMajor(from: "5.10.1"))
```

Comment:
No needs to change that, you just need to remove Package.resolved and open the Xcode and it will download the latest version (as the major version wasn't bumped).